apps/src/bin/quiche-server.rs

```diff
@@ -61,22 +61,19 @@ fn main() {
     let args = ServerArgs::with_docopt(&docopt);
 
     // Setup the event loop.
-    let poll = mio::Poll::new().unwrap();
+    let mut poll = mio::Poll::new().unwrap();
     let mut events = mio::Events::with_capacity(1024);
 
     // Create the UDP listening socket, and register it with the event loop.
     let socket = net::UdpSocket::bind(args.listen).unwrap();
+    socket.set_nonblocking(true).unwrap();
```

Comment:
Do we need this?

Reply:
Without this line, the socket will blocks on the recv_from call below (as far as I confirmed in my local env): https://github.com/cloudflare/quiche/blob/0107fb90694d0144bcf76a125c67abb84e2fe204/apps/src/bin/quiche-server.rs#L170

Please also see the comment in the mio repo: https://github.com/tokio-rs/mio/blob/b2bbc1029849c939031fb396475d1a19da9786a6/tests/udp_socket.rs#L102-L105